### PR TITLE
A300

### PR DESCRIPTION
**Pull request by roni-kreinin**

### CODE REVIEW

File: clearpath_hardware_interfaces/include/clearpath_hardware_interfaces/lighting/color.hpp [hunk 1]
```diff
@@ -62,18 +62,18 @@ struct hsv_t {
...
-static const hsv_t COLOR_BLUE = hsv_t(240.0, 100.0, 100.0);
-static const hsv_t COLOR_BLUE_DIM = hsv_t(240.0, 100.0, 10.0);
-static const hsv_t COLOR_CYAN = hsv_t(180.0, 100.0, 100.0);
-static const hsv_t COLOR_GREEN = hsv_t(120.0, 100.0, 100.0);
-static const hsv_t COLOR_GREEN_DIM = hsv_t(120.0, 100.0, 10.0);
-static const hsv_t COLOR_YELLOW = hsv_t(60.0, 100.0, 100.0);
-static const hsv_t COLOR_ORANGE = hsv_t(30.0, 100.0, 100.0);
-static const hsv_t COLOR_WHITE = hsv_t(0.0, 0.0, 100.0);
-static const hsv_t COLOR_WHITE_DIM = hsv_t(0.0, 0.0, 10.0);
+static const hsv_t COLOR_RED = hsv_t(0.0, 100.0, 37.5);
```
Comment (roni-kreinin) [inline clearpath_hardware_interfaces/include/clearpath_hardware_interfaces/lighting/color.hpp:65]: We will need to test these values on other robots and adjust as necessary

**Review by luis-camero — APPROVED** [review]

File: clearpath_hardware_interfaces/include/clearpath_hardware_interfaces/lighting/color.hpp (hunk 1 above)
Comment (civerachb-cpr) [reply clearpath_hardware_interfaces/include/clearpath_hardware_interfaces/lighting/color.hpp:65]: Is there a ticket somewhere to track that testing so it doesn't get lost/forgotten about?

Comment (roni-kreinin) [reply clearpath_hardware_interfaces/include/clearpath_hardware_interfaces/lighting/color.hpp:65]: I'll make one

Comment (civerachb-cpr) [reply clearpath_hardware_interfaces/include/clearpath_hardware_interfaces/lighting/color.hpp:65]: Thanks!

Comment (roni-kreinin) [reply clearpath_hardware_interfaces/include/clearpath_hardware_interfaces/lighting/color.hpp:65]: RPSW-2399

**Review by civerachb-cpr — APPROVED** [review]
Tested all packages locally; they pass. I suspect the CI is failing because of synchronization issues across repos (i.e this test ran before another repo's MR was merged).

Everything seems okay to me.

**Pull request merged**